tests/unittests/test_handler/test_handler_reset_rmc.py

```diff
@@ -0,0 +1,42 @@
+from cloudinit.config import cc_reset_rmc
+
+from cloudinit import cloud
+from cloudinit import helpers
+from cloudinit import util
+
+from cloudinit.tests import helpers as t_help
+
+import logging
+import os
+
+LOG = logging.getLogger(__name__)
+
+
+class TestRsctNodeFile(t_help.FilesystemMockingTestCase):
+    def setUp(self):
+        super(TestRsctNodeFile, self).setUp()
+        self.tmp = self.tmp_dir()
+
+    def cc_cloud(self):
+        paths = helpers.Paths({})
+        ds = None
+        return cloud.Cloud(ds, paths, {}, None, None)
+
+    def rsct_node(self):
```

Comment:
unit tests should not run as root.  so we can't expect to remove or write to the file /etc/ct_node_id.

You'll have to mock or otherwise adjust code to test.